src/Block.cpp

```diff
@@ -426,7 +426,8 @@ Block::append_return_stmt(CGContext& cg_context)
 	ERROR_GUARD(NULL);
 	stms.push_back(sr);
 	fm->makeup_new_var_facts(pre_facts, fm->global_facts);
-	assert(sr->visit_facts(fm->global_facts, cg_context));
+	bool visited = sr->visit_facts(fm->global_facts, cg_context);
+	assert(visited);
```

Comment:
By default release build has no assertions (I changed that for VS in a later commit). In that case this change is needed to produce absolutely same code by release csmith and debug csmith. This is helpful to be able to debug.
I found ~3 such asserts about a year ago; didn't double-check this now.